### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by 8i11y**
Deleted all the entries I don't have on my oxid eshop 6.2 installation (maybe ghostfiles / -directories from older versions !? )

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,24 +26,6 @@
...
 /composer.lock
 
-# Testing library
-/test_config.yml
-/source/Services/
-
-# Editions
-/source/Edition/
-
-# IDE
```
Comment (Sieg) [inline .gitignore:36]: IDE section is used, it ignores everything IDE is generating

File: .gitignore [hunk 2]
```diff
@@ -26,24 +26,6 @@
...
-/.idea/
-/.phpstorm-oxid.meta.php
-/.ide-helper.php
-
-# OS
```
Comment (Sieg) [inline .gitignore:41]: Mac os is generating those files, should be ignored as well

Comment (8i11y) [reply .gitignore:41]: ok - so these are all things which are generated in some cases
Didn't know if these are already needed or remained from older versions of oxid ^^
so I wrote it here as a pull request as a hint and you have a look at it ;)

**Comment by Sieg** [comment]
Hey @8i11y, thanks for triggering this and your suggested changes. I have reverted the part that is still used and merged the PR. Have a nice day!

**Comment by 8i11y** [comment]
thx Sieg, have also a nice day :)

**Pull request merged**